src/kvirc/ui/KviMenuBar.h

```diff
@@ -56,6 +56,7 @@ class KVIRC_API KviMenuBar : public KviTalMenuBar
 	QMenu                       * m_pActionsToolsPopup;
 	KviPointerList<KviScriptMenuBarItem> * m_pScriptItemList;
     // Dynamic actions
+    QAction                     * m_pMenuBarAction;
```

Comment:
small nitpick, can we fix the indenting on separate commit for this file while we are here?